Content.Server/Station/Systems/StationSystem.cs

```diff
@@ -148,6 +149,16 @@ private void OnRoundEnd(GameRunLevelChangedEvent eventArgs)
         }
     }
 
+    private void OnStationGridDeleted(EntityUid uid, StationMemberComponent component, ComponentShutdown args)
+    {
+        if (Deleted(component.Station))
+            return;
+
+        var data = Comp<StationDataComponent>(component.Station);
```

Comment:
Shouldn't this trycomp.

Reply:
All stations must have StationDataComponent to function, period, so this isn't needed.

Reply:
What if I delete the station entity.